src/main/java/pageclasses/MailPage.java

```diff
@@ -1,10 +1,17 @@
 package pageclasses;
 
-import org.openqa.selenium.By;
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
+import java.io.File;
+import java.io.IOException;
+
 public class MailPage extends BasePage {
+    private static final String SCREENSHOT_PATH = "src/test/resources/mail.png";
```

Comment:
if you want to have different screenshots for each test run, it is better to bind the file name to the date + time, for example.